src/dump_sources.rs

```
@@ -0,0 +1,485 @@
+use std::env;
+use std::ffi::OsString;
```

Clippy (1.96.0)

[warning] unused import

(warning)